apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/Indictment.tsx

```diff
@@ -37,13 +40,24 @@ const Indictment: React.FC = () => {
   } = useContext(FormContext)
   const { formatMessage } = useIntl()
   const { updateCase, setAndSendCaseToServer } = useCase()
-  const stepIsValid = true
+  const stepIsValid = isTrafficViolationStepValidIndictments(workingCase)
```

Comment:
Ah, reminds me I need to add validation to the police case number field on my branch 👍